modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -358,12 +399,23 @@ if [ -n "$FSIMG" ]; then
     fi
     if [ "$FSIMG" = "$SQUASHED" ]; then
         BASE_LOOPDEV=$SQUASHED_LOOPDEV
+        if [ -n ${join} ]; then
```

Comment:
Device-mapper based configurations will not support a union join of rootfs content with joinfs.  This code is superfluous, no?